chrome/background.entry.js

```diff
@@ -162,6 +162,12 @@ addListener('addURLToHistory', url => {
 	chrome.history.addUrl({ url });
 });
 
+addListener('getVisitedURLs', async urls =>
+	(await Promise.all(
+		urls.map(async url => [url, await apiToPromise(chrome.history.getVisits)({ url })])
+	)).filter(v => v[1].length).map(v => v[0])
```

Comment:
If this is returning multiple results, I'd prefer a map from url -> visited state instead of an array of visited urls.

Although, since it's only ever used to check one URL at a time (AFAICT), you could just accept a single URL and return a boolean, which would simplify the implementation a bit. We can always expand it to accept multiple URLs in the future if necessary.